<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="dlgBtn_set_room_data">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Raumbezeichnung:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_target_temp">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Solltemperatur [C]:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_target_humidity">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sollfeuchtigkeit [%]:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_target_temp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_target_humidity">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
